tasks/xpack/security/elasticsearch-xpack-activation.yml

```diff
@@ -9,7 +9,7 @@
     return_content: yes
   register: license_activated
   no_log: True
-  when: not "security" in es_xpack_features
```

Comment:
This is another task where we could use omit to combine them into a single task. Not blocking for this PR though.